research/conditional/utils/conditional_trainer.py

```diff
@@ -370,11 +404,12 @@ def _decode_samples(self, step):
                 iteration=step,
             )
```

Comment:
I don't like abbreviating `batch_size` to `bsz`

Reply:
ok, I changed it